examples/workflow/2_generate_data.ipynb

```diff
@@ -50,18 +50,20 @@
    "source": [
     "1. Import the section of particle definitions that is embedded in the XML file.\n",
     "\n",
-    "2. Build a :class:`~.Kinematics` object following the specifications of the\n",
+    "2. Build a :class:`.Kinematics` object following the specifications of the\n",
     "   dynamics in the XML model file (in this case, the helicity formalism).\n",
     "   The second argument of this function is a particle list.\n",
     "\n",
-    "3. Use this :class:`~.Kinematics` object to generate a phase space sample as\n",
-    "   well. This sample defines the space on which to evaluate the intensities\n",
-    "   of the amplitude model.\n",
+    "3. The information in the :class:`.Kinematics` instance is sufficient for\n",
+    "   generating a phase space sample. This sample defines the space on which to\n",
+    "   evaluate the intensities of the amplitude model.\n",
     "\n",
-    "4. Build an :class:`~.Intensity` object from the amplitude model. Because we\n",
-    "   want the intensities to be normalized, you have to pass the intensity\n",
-    "   builder function a phase space sample as an argument. Note that during the\n",
-    "   creation of the :class:`~.Intensity` object, the :class:`~.Kinematics`\n",
+    "4. For generating a data sample, you require an :class:`.Intensity` profile\n",
+    "   for this specific decay. You can construct such an object from the model\n",
+    "   XML file using the :class:`.IntensityBuilderXML`. Because we want the\n",
+    "   intensities to be normalized, you have to pass this intensity builder\n",
+    "   function the generated phase space sample as an argument. Note that during\n",
```

Comment:
you have to pass this intensity builder function the generated phase space sample as an argument
-> you have to pass the generated phase space sample as an argument to the intensity builder

Reply:
https://github.com/ComPWA/pycompwa/pull/79/commits/63dec07f2057995445763d6ea7dba02c3dea338d